### ISSUE TITLE
BUG. Workrave does not detect inactivity on Windows 10 after Windows update.

### ISSUE THREAD

**Issue by MichaelOmegatron** [rcaelers/workrave#229]
**Describe the bug**
1. Workrave does not detect my inactivity. When a Micro Break or Rest Break appears, the timer does not count down past 2 seconds into the break, because it does not detect my inactivity. Even if I turn off my keyboard and mouse. So if I have the "Block option" set to "No Input" it effectively locks me out of my computer.

**To Reproduce**
1. Opening Workrave (possibly after the Windows 10 update automatically downloaded on 9/16/2020. OS Build 19041.508)
2. Don't input any commands with keyboard or mouse. Remain idle and inactive.
3. See error: Workrave does not detect inactivity. The blue Active Bar continues to fill up. Micro Breaks and Rest Breaks get stuck and fail to proceed with their break time.

**Expected behavior**
When I do not use my mouse or keyboard, I expect Workrave to detect the inactivity so that breaks can go through their cycle and not get stuck in an endless break.

** Windows (please complete the following information in case you encountered the bug on Windows):
 - Windows Version: Windows 10 Pro. OS Build: 19041.508 Windows Feature Experience Pack 120.2212.31.0
 - Workrave Version: 1.10.44

**Additional context**
Early in the morning on 9/16/2020 my computer needed to restart to complete some automatic updates for Windows 10. I noticed that Workrave (some older version) wasn't detecting my inactivity. So I uninstalled it and reinstalled it with the newest version (1.10.44). The bug still occurs however. I know my friend had an issue with her time clock being incorrect after some recent Windows Update. And some data from a game I was playing before the update was lost. I get the impression that this windows update messed some stuff up, and I'm wondering if Workrave might have been affected.

**Comment by utkuerd** [comment]
I'm having the same issue, though Windows is even having other issues about inactivity too. For example my computer did not go to sleep or turn off the screen ever since I've done the update.

**Comment by Stargrazer82301** [comment]
I get this too, and it also started after a Windows 10 update reboot.

**Comment by piperfan** [comment]
Same here. Micro & rest break stops 2 seconds in. Not working in lock screen either.  Enabling alternate monitor did not make any difference.

**Comment by MichaelOmegatron** [comment]
I'm happy to say that this problem seems to have been resolved for me! A user here mentioned their computer no longer going into sleep mode after the Windows update I mentioned above. I realized this was true for me too. My computer would not go into sleep mode. This made me suspect this was a Windows problem, and not a Workrave issue. Recently my computer downloaded a Windows 10 update (**OS Build: 19041.572**) and now my computer goes into sleep mode. I tested out Workrave, and it now behaves as expected. It seems like Windows wasn't detecting idle time. But now that it does, so does Workrave.
